<ui version="4.0" >
 <class>KWin::ThumbnailAsideEffectConfigForm</class>
 <widget class="QWidget" name="KWin::ThumbnailAsideEffectConfigForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Appearance</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Maximal &amp;width:</string>
        </property>
        <property name="buddy" >
         <cstring>spinWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="spinWidth" >
        <property name="suffix" >
         <string> px</string>
        </property>
        <property name="maximum" >
         <number>500</number>
        </property>
        <property name="value" >
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>&amp;Spacing:</string>
        </property>
        <property name="buddy" >
         <cstring>spinSpacing</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="spinSpacing" >
        <property name="suffix" >
         <string> px</string>
        </property>
        <property name="maximum" >
         <number>30</number>
        </property>
        <property name="value" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>&amp;Opacity:</string>
        </property>
        <property name="buddy" >
         <cstring>spinOpacity</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="spinOpacity" >
        <property name="suffix" >
         <string> %</string>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KWin::GlobalShortcutsEditor" native="1" name="editor" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KWin::GlobalShortcutsEditor</class>
   <extends>QWidget</extends>
   <header location="global" >kwineffects.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
